### PR TITLE
Added combobox virtualizer story

### PR DESCRIPTION
**Pull request by GeoffCoxMSFT**
## Changes
- Added story to Combobox to demonstrate using Virtualizer and displaying 10K options.

### CODE REVIEW

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29763/perf-test-react-components/Avatar-mount_base.html">621</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29763/perf-test-react-components/Avatar-mount.html">646</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29763/perf-test-react-components/Button-mount_base.html">325</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29763/perf-test-react-components/Button-mount.html">317</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29763/perf-test-react-components/Field-mount_base.html">1120</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29763/perf-test-react-components/Field-mount.html">1072</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29763/perf-test-react-components/FluentProvider-mount_base.html">709</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29763/perf-test-react-components/FluentProvider-mount.html">697</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29763/perf-test-react-components/FluentProviderWithTheme-mount_base.html">81</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29763/perf-test-react-components/FluentProviderWithTheme-mount.html">76</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29763/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">63</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29763/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">61</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29763/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">93</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29763/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">69</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29763/perf-test-react-components/MakeStyles-mount_base.html">880</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29763/perf-test-react-components/MakeStyles-mount.html">875</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29763/perf-test-react-components/Persona-mount_base.html">1713</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29763/perf-test-react-components/Persona-mount.html">1660</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29763/perf-test-react-components/SpinButton-mount_base.html">1359</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29763/perf-test-react-components/SpinButton-mount.html">1355</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Combobox.fixture.js'>Combobox (including child components)</abbr>                                                 |  `92.033 kB`<br />`30.008 kB` |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Dropdown.fixture.js'>Dropdown (including child components)</abbr>                                                 |  `90.475 kB`<br />`29.668 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr>              |   `70.013 kB`<br />`20.17 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `207.423 kB`<br />`59.298 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `42.297 kB`<br />`14.008 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>                                                         |    `6.651 kB`<br />`2.252 kB` |
| <samp>react-timepicker-compat-preview</samp> <br /> <abbr title='bundle-size/TimePicker.fixture.js'>TimePicker</abbr>                                                         |  `93.434 kB`<br />`31.284 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/01a8b6993157ff5f76326479a81fc9e9a70864f8'>01a8b6993157ff5f76326479a81fc9e9a70864f8</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

File: packages/react-components/react-combobox/stories/Combobox/ComboboxVirtualizer.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  });
+
+  const styles = useStyles();
+
+  return (
+    <div>
+      <div>
+        <label htmlFor={`${comboId}`}>Medium</label>
+        <Combobox
+          aria-labelledby={`${comboId}`}
```
Comment (smhigley) [inline packages/react-components/react-combobox/stories/Combobox/ComboboxVirtualizer.stories.tsx:34]: small nit, this should actually be `id` (or alternatively, keep the `aria-labelledby` here and put `id` on the label, but I'd prefer the native htmlFor/id pair)

```suggestion
          id={`${comboId}`}
```

**Review by smhigley — APPROVED** [review]

**Pull request merged**